modules/claims_api/app/controllers/claims_api/v2/veterans/power_of_attorney/request_controller.rb

```diff
@@ -30,6 +30,34 @@ def index
           render json: poa_list, status: :ok
         end
 
+        def decide
+          proc_id = form_attributes['procId']
+
+          unless proc_id
+            raise ::Common::Exceptions::ParameterMissing.new('procId',
+                                                             detail: 'procId is required in the URL')
```

Comment:
@tycol7 Just one question, does this error message make sense here?  Is the procId sent as a param in the URL?

I just wasn't sure if 
```
We can pass in the proc_id in the URL or in the request body.
```

Meant the solution is either/or, or if one had been decided upon, so just looking for clarification.  Otherwise this performed as expected and looked good.

Reply:
Oh, good catch. It's supposed to be in the body. Either works, but since we have to send in a `decision` anyway, I figured why not `procId` too. Will update the error detail.

Reply:
https://github.com/department-of-veterans-affairs/vets-api/pull/18864/commits/cc7ad765755b944e267b30ee62b30f38d9efb1fe